hazelcast-spring/src/main/resources/hazelcast-spring-3.9.xsd

```diff
@@ -1471,7 +1471,7 @@
         <xs:complexType>
             <xs:complexContent>
                 <xs:extension base="hazelcast-bean">
-                    <xs:all>
+                    <xs:choice minOccurs="1" maxOccurs="unbounded">
```

Comment:
Okay, each child element defines its own `minOccurs="0" maxOccurs="1"` etc., so this seems to override the `<xs:choice minOccurs="1" maxOccurs="unbounded">`. But I still don't understand the meaning of the `minOccurs` then.